### DISCUSSION TITLE
Sign My Petition Against Mandatory 2FA

### DISCUSSION THREAD

**Discussion by Chameleon-Coder** [community/community#146085] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Feature Area

Projects

### Body

> URL hidden by staff

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by shinybrightstar** [comment]
Hi @Chameleon-Coder 

Thanks for reaching out in GitHub's Community! 
Any links shared in GitHub Community discussions should be shared with the intent of providing relevant and appropriate information. Links should not be posted to simply drive traffic or attention to a site. Links should always be accompanied by a full explanation of the content and purpose of the link. Posting links, especially unsolicited ones, without relevant and valuable context can come across as advertising or serving even more malicious purposes.

Please review our [Code of Conduct]( https://docs.github.com/en/site-policy/github-terms/github-community-code-of-conduct). This type of post is considered "off topic" for our Community Discussions and will be closed. Any repeat and future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.

**Comment by MasterInQuestion** [comment]
Security theater on display:
    https://github.com/orgs/community/discussions/139621#discussioncomment-11125160
    (on-device one-liner TOTP)

    Though pointlessly silly, it should workaround the bother.

    “I used to use GitHub, but then they started enforcing multi-factor authentication on me.
    This happened to other people as well.”
> URL hidden by staff
